modules/install-roxctl-cli-linux.adoc

```diff
@@ -9,11 +9,12 @@ You can install the `roxctl` CLI binary on Linux by using the following procedur
 
 .Procedure
 
-. Download the latest version of the `roxctl` CLI:
+. Download the `roxctl` CLI (supported architectures are `amd64`, `ppc64le` and `s390x`):
 +
 [source,terminal,subs=attributes+]
 ----
-$ curl -O https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Linux/roxctl
+$ arch="$(uname -m | sed "s/x86_64//")"; arch="${arch:+-$arch}"
+$ curl -f -o roxctl "https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Linux/roxctl${arch}"
```

Comment:
```suggestion
[source,terminal,subs=attributes+]
----
$ arch="$(uname -m | sed "s/x86_64//")"; arch="${arch:+-$arch}"
----
+
[source,terminal,subs=attributes+]
----
$ curl -f -o roxctl "https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Linux/roxctl${arch}"
----
```

Reply:
I think we can break this into 2 separate code blocks for better readability

Reply:
Makes sense, done.